quilt/tools/main.py

```diff
@@ -42,6 +42,8 @@ def main():
 
     push_p = subparsers.add_parser("push")
     push_p.add_argument("package", type=str, help="Owner/Package Name")
+    push_p.add_argument("--public", action="store_true",
+                        help="Create a public package (not valid if the package already exists")
```

Comment:
I wanted to emphasize "create" cause there's no point in using it for existing packages. But sure, I'll change it.